app/ai/gemini.py

```
@@ -0,0 +1,88 @@
+
+
+def _configure() -> None:
+    global _configured
+    if _configured:
+        return
+    api_key = os.getenv("GEMINI_API_KEY") or os.getenv("GEMINI_API_KEY")
+    if not api_key:
+        # Defer strict error so the app can start without LLM usage
+        return
+    if genai is None:
+        raise RuntimeError("google-generativeai package not installed; add to requirements.txt")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)